build.gradle

```diff
@@ -18,6 +18,9 @@ buildscript {
         classpath "org.jacoco:org.jacoco.core:${versions.jacocoGradlePlugin}"
         classpath "org.jetbrains.kotlinx:atomicfu-gradle-plugin:${versions.atomicFuPlugin}"
         classpath "com.vanniktech:gradle-maven-publish-plugin:${versions.mavenPublishPlugin}"
+        classpath("app.cash.zipline:zipline-gradle-plugin:0.9.1")
+        classpath("com.squareup.sqldelight:gradle-plugin:1.5.3")
+        classpath("com.google.dagger:hilt-android-gradle-plugin:2.42")
```

Comment:
in this house we use anvil :-D jk, glad you got this into the sample, extract version pls